tekton/resources/images/ko-multi-arch-template.yaml

```
@@ -77,8 +77,9 @@ spec:
           script: |
             #!/usr/bin/env sh
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot
+            IMAGE_REGISTRY_TOKEN=$(cat "$IMAGE_REGISTRY_CREDENTIALS")
+            docker login -u ${IMAGE_REGISTRY_USER} --password-stdin ${IMAGE_REGISTRY_TOKEN}
```

When using `--password-stdin` the password must be provided via standard input, e.g.

`echo PASSWORD | docker login -u USERNAME --password-stdin`

or

`docker login -u USERNAME --password-stdin < /path/to/file/containing/PASSWORD`

You'll also need to specify the registry, otherwise `docker login` will default to attempting to authenticate against Docker Hub.


```suggestion
            echo "$IMAGE_REGISTRY_TOKEN" | docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin
```